### DISCUSSION TITLE
Setting Primary and Secondary Color via API

### DISCUSSION THREAD

**Discussion by RobAuto87** [etsy/open-api#1141] [Q&A]
Hi All, 

I have search everywhere but I can seem to find any documentation on how to set the Primary and Secondary color for a listing.

I can find the taxonomies using getSellerTaxonomyNodes. But this doesn't have a primary and secondary color option.

The only instance I can find is found here https://developers.etsy.com/documentation/tutorials/listings#how-to-fetch-property-values-for-the-products-array

Under Notes about taxonomy
_"Some of the common taxonomy node properties that Etsy uses, such as Color have a list of common values/value ids. When you pass in value_id 4 we convert that to your shop's unique ID for that color."_

![2023-10-23 14_39_32-Edit _ First Product Test Green  G009](https://github.com/etsy/open-api/assets/58709026/0ca05977-dece-4798-ba9e-9a72b4e0688f)

**Comment by leedyche** [comment]
did you find an anwer to this? thanks

**Comment by showtroylove** [comment]
Use the listing's taxonomy id to lookup its associated properties (i.e. Primary & Secondary color, among others).  The following API's are required:

### Taxonomy (Category) Properites
**getPropertiesByTaxonomyId**

### Listing Properties (CRUD)
**getListingProperties**
**updateListingProperty**
..

**Be sure to mark this comment as an answer if it helps you resolve your question.**
